docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -0,0 +1,98 @@
+``chgres_cube``
+===============
+
+The ``uw`` mode for configuring and running the :ufs-utils:`chgres_cube<chgres-cube>` component.
+
+.. code-block:: text
+
+   $ uw chgres_cube --help
+   usage: uw chgres_cube [-h] TASK ...
+
+   Execute chgres_cube tasks
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+
+   Positional arguments:
+     TASK
+       namelist_file
+         The namelist file
+       provisioned_run_directory
+         Run directory provisioned with all required content
+       run
+         A run
+       runscript
+         The runscript
+
+
+All tasks take the same arguments. For example:
+
+.. code-block:: text
+
+   $ uw chgres_cube run --help
+   usage: uw chgres_cube run --config-file PATH --cycle CYCLE [-h] [--batch] [--dry-run]
+                             [--graph-file PATH] [--quiet] [--verbose]
+
+   A run
+
+   Required arguments:
+     --config-file PATH, -c PATH
+         Path to config file
+     --cycle CYCLE
+         The cycle in ISO8601 format
+
+   Optional arguments:
+     -h, --help
+         Show help and exit
+     --batch
+         Submit run to batch scheduler
+     --dry-run
+         Only log info, making no changes
+     --graph-file PATH
+         Path to Graphviz DOT output [experimental]
+     --quiet, -q
+         Print no logging messages
+     --verbose, -v
+         Print all logging messages
+
+
+Examples
+^^^^^^^^
+
+The examples use a configuration file named ``config.yaml`` with content similar to:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/chgres_cube.yaml
+
+
+Its contents are described in depth in section :ref:`chgres_cube_yaml`.
+
+* Run ``chgres_cube`` on an interactive node
+
+  .. code-block:: text
+
+     $ uw chgres_cube run --config-file config.yaml --cycle 2023-12-15T18
+
+  The driver creates a ``runscript`` file in the directory specified by ``run_dir:`` in the config and runs it, executing ``chgres_cube``.
+
+* Run ``chgres_cube`` via a batch job
+
+  .. code-block:: text
+
+     $ uw chgres_cube run --config-file config.yaml --cycle 2023-12-15T18 --batch
+
+  The driver creates a ``runscript`` file in the directory specified by ``run_dir:`` in the config and submits it to the batch system. Running with ``--batch`` requires a correctly configured ``platform:`` block in ``config,yaml``, as well as appropriate settings in the ``execution:`` block under ``chgres_cube:``.
```

Comment:
```
...requires a correctly configured ``platform:`` block in ``config,yaml``
```
change ``config,yaml`` to ``config.yaml``. Looks like this needs to be updated in the sfc_climo_gen docs as well.